src/components/calcite-block/calcite-block.scss

```diff
@@ -21,138 +28,147 @@
 }
 
 .header-container {
-  display: grid;
+  @apply grid items-stretch;
   grid-template: auto / auto 1fr auto;
   grid-template-areas: "handle header control";
   grid-column: header-start / control-end;
   grid-row: 1 / 2;
-  align-items: stretch;
 
   & > .header {
-    padding: var(--calcite-spacing-three-quarters) 0;
+    @apply py-3 px-0;
   }
 }
 
 .toggle {
-  display: flex;
-  align-items: center;
-  flex-wrap: nowrap;
-  margin: 0;
-  padding: var(--calcite-spacing-three-quarters) 0;
-  background-color: transparent;
-  border: none;
-  cursor: pointer;
-  font-family: inherit;
-  text-align: unset;
-  @include focus-style-base();
+  @apply
+    bg-transparent
+    flex
+    flex-no-wrap
+    font-inherit
+    items-center
+    m-0
+    py-3
+    px-0
+    border-none
+    cursor-pointer
+    text-left
+    focus-style-base;
+
   &:hover {
-    background-color: var(--calcite-ui-foreground-2);
+    @apply bg-foreground-2;
   }
   &:focus {
-    @include focus-style-inset();
+    @apply focus-style-inset;
   }
 }
 
 calcite-loader[inline] {
   grid-area: control;
-  align-self: center;
+  @apply self-center;
 }
 
 calcite-handle {
   grid-area: handle;
 }
 
 .title {
-  margin: 0;
-  padding: 0 var(--calcite-spacing-three-quarters);
+  @apply m-0 px-3 py-0;
 }
 
 .header .title .heading {
-  padding: 0;
-  color: var(--calcite-ui-text-3);
-  transition: color $transition;
-  font-size: theme("fontSize.-1");
-  @include word-break();
+  @apply
+    p-0
+    text--1
+    text-color-3
+    word-break
+    transition-color
+    duration-150
+    ease-in-out;
 }
 
 .summary {
-  color: var(--calcite-ui-text-3);
-  padding: 0;
-  font-size: theme("fontSize.-2");
-  @include word-break();
+  @apply
+    p-0
+    text--2
+    text-color-3
+    word-break
 }
 
 .icon {
-  margin-left: var(--calcite-spacing-three-quarters);
+  @apply ml-3;
 }
 
 .toggle-icon {
-  fill: currentColor;
-  flex: 0 0 var(--calcite-icon-size);
-  margin: 0 var(--calcite-spacing) 0 0;
+  @apply
+    flex-grow-0
+    flex-shrink-0
+    fill-current
+    m-0
+    mr-4;
+
+  flex-basis: theme("spacing.4");
```

Comment:
☝️ is there an equivalent tailwind class for this flex basis rule?

Reply:
I didn't see one.
I think Tailwind assumes the browser will do the right thing...but I haven't always found that to be the case.